projects/plugins/jetpack/modules/contact-form/admin.php

```diff
@@ -832,6 +872,7 @@ function grunion_enable_spam_recheck() {
 	}
 
 	// Add the actual "Check for Spam" button.
+	add_action( 'admin_head', 'grunion_export_button' );
```

Comment:
Because of the validation above:

```
	if ( ! defined( 'AKISMET_VERSION' ) ) {
		return;
	}
```

The Export button is only visible if the Akismet plugin is active.
Was that intentional?

Reply:
Nice catch, fixed 👍